item-spec/item-spec.md

```diff
@@ -77,27 +77,34 @@ datetime fields.
 
 This object describes a relationship with another entity. Data providers are advised to be liberal
 with the links section, to describe things like the catalog an item is in, related items, parent or
-child items (modeled in different ways, like an 'acquisition' or derived data). The `self` link is
-required as an absolute URL, to represent the location that the Item can be found online. It is
-allowed to add additional fields such as a `title` and `type`.
+child items (modeled in different ways, like an 'acquisition' or derived data).
+It is allowed to add additional fields such as a `title` and `type`.
 
 | Field Name | Type   | Description                                                                                                                         |
 | ---------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------- |
-| href       | string | **REQUIRED.** The actual link in the format of an URL. Relative and absolute links are both allowed.                                |
+| href       | string | **REQUIRED.** The actual link in the format of an URL. Relative and absolute links are both allowed unless otherwise noted in the ```rel``` type description.                    |
 | rel        | string | **REQUIRED.** Relationship between the current document and the linked document. See chapter "Relation types" for more information. |
 | type       | string | Media type of the referenced entity. 																								|
 | title      | string | A human readable title to be used in rendered displays of the link. |
 
+#### Relative vs Absolute links
+
+Currently the JSON schema for links does not require a URI formatting, to give the option for
```

Comment:
```suggestion
Currently, the JSON schema for links does not require them to be formatted as URIs, to allow
```

Reply:
It sounds like we should be able to use another JSON construct (uri-ref or something? @m-mohr was talking about this) so the schema can validate relative and absolute links.